robolectric/src/test/java/org/robolectric/shadows/ShadowDisplayTest.java

```diff
@@ -171,8 +171,8 @@ public void setDisplayHdrCapabilities_shouldReturnHdrCapabilities() {
   }
 
   @Test
-  @Config(minSdk = M)
-  public void setDisplayHdrCapabilities_shouldThrowUnSupportedOperationException() {
+  @Config(maxSdk = M)
```

Comment:
Can the changes to ShadowDisplayTest be put in a different PR to keep history clean? Seems like it is a little more than just error prone cleanup.

Reply:
Sure, done.